content/en/docs/how-tos/testing-operator-sdk-operators.md

```diff
@@ -185,7 +191,7 @@ tests:
       OO_PACKAGE: "myoperator"
       OO_TARGET_NAMESPACES: '!install'
     dependencies:
-      OO_INDEX: "ci-index-my-bundle" # if the bundle being tested is named, update the dependency to match
+      OO_BUNDLE: "bundle-name" # if the name of the bundle being tested is not my-bundle in the ci-opeator's configuration, update the dependency to match
```

Comment:
Should it be the following as that is the default?
```suggestion
      OO_BUNDLE: "my-bundle" # if the name of the bundle being tested is not my-bundle in the ci-opeator's configuration, update the dependency to match
```

Reply:
It is how the dependency feature works: default value is defined in the step and can be overwritten by test.
Here is the overwritten part.

It is similar to `OO_INDEX` in the previous work: by default, it is `ci-index`. The test overwrites its value by `ci-index-my-bundle` which is the tag name of the index image for a named bundle.